src/components/GraphiQL.js

```diff
@@ -678,7 +689,11 @@ export class GraphiQL extends React.Component {
         operationName,
         ...queryFacts
       });
+      
+      return operationName;
     }
+    
+    return null;
```

Comment:
This line isn't necessary, by default returning undefined will behave as expected